docker.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/hex"
+	"os"
+	"path/filepath"
+
+	"github.com/fsouza/go-dockerclient"
+
+	c "github.com/wantedly/risu/cache"
+	"github.com/wantedly/risu/schema"
+)
+
+const (
+	DefaultDockerEndpoint = "unix:///var/run/docker.sock"
+)
+
+func dockerBuild(build schema.Build) error {
+	clonePath := DefaultSourceBaseDir + build.SourceRepo
+	cache := c.NewCache(os.Getenv("CACHE_BACKEND"))
```

Comment:
How about extract these 22~44 lines to `addCacheToSrcRepo()`.
I think it can be more clear that what this lines is doing.